### ISSUE TITLE
B.2.6.500 - Adding and Removing Users from Roles

### ISSUE THREAD

**Issue by aldefouw** [4bbakers/redcap_rsvc#43]
Feature: [B.2.6.500 - Adding and Removing Users from Roles](https://github.com/4bbakers/redcap_rsvc/tree/v14.7.0/./Feature%20Tests/B/Assign%20User%20Rights_6/B.2.6.500%20-%20Adding%20and%20Removing%20Users%20from%20Roles.feature)

**Comment by MintooXavier** [comment]
@aldefouw : The following step is not working. 
I click on the button labeled "Assign to role" in the role selector dropdown

**Comment by mmcev106** [comment]
@aldefouw, even after including the changes from #164, I'm still seeing a [failure on this issue](https://cloud.cypress.io/projects/pvu79o/runs/1966/test-results/5f6b0520-0191-405b-a996-34119c14f67f?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%7B%22label%22%3A%22redcap_rsvc%2FFeature%20Tests%2FB%2FAssign%20User%20Rights_6%2FB.2.6.500%20-%20Adding%20and%20Removing%20Users%20from%20Roles.feature%22%2C%22value%22%3A%22%5B%5C%22594bd5d7-66e5-4513-ae20-7f79e1c005a8%5C%22%5D%22%7D%5D&statuses=%5B%5D&testingTypesEnum=%5B%5D).  Do you know why that might be?

**Comment by aldefouw** [comment]
@mmcev106 - seems to be working in your [latest run](https://cloud.cypress.io/projects/pvu79o/runs/2023/test-results/18844eb2-7157-4506-bb80-7977f16d070f/stdout?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%7B%22label%22%3A%22redcap_rsvc%2FFeature%20Tests%2FB%2FAssign%20User%20Rights_6%2FB.2.6.500%20-%20Adding%20and%20Removing%20Users%20from%20Roles.feature%22%2C%22value%22%3A%22%5B%5C%2241bf2a81-3566-43ec-bc9a-0891595b4cac%5C%22%5D%22%7D%5D&statuses=%5B%5D&testingTypesEnum=%5B%5D) from what I can see?

**Comment by mmcev106** [comment]
Indeed!  I'll go ahead and close this, then go through our swim lanes to see if recent rctf update fixed anything else.  Thanks for the heads up!
